code/modules/surgery/rig_removal.dm

```diff
@@ -1,59 +1,45 @@
 //Procedures in this file: Unsealing a Rig.
```

Comment:
I think this entire file will be deleted when you merge upstream, because of the rig removal.